dags/hivemind_etl_helpers/tests/integration/test_gdrive_loader.py

```diff
@@ -189,8 +187,8 @@ def test__load_by_folder_id(self, mock_load_data):
         self.mock_loader = Mock()
         mock_load_data.return_value = mock_data
         loader = GoogleDriveLoader(client_config=self.mock_client_config)
-        result = loader._load_from_folders(folder_ids=folder_id)
-        self.assertEqual(len(result), 4)
+        result = loader._load_from_folders(folder_id=folder_id)
```

Comment:
if you're accepting a string as `file_id` then passing the list here is not the right solution. Please first correct the logic of your `_load_from_folders` method (based on the previous comment) and then if required update the test cases.